net/tests/Sails.TestUtils/XUnit/Fixtures/SailsFixture.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.IO;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Nito.AsyncEx;
+using Sails.TestUtils.Containers;
+using Sails.TestUtils.Git;
+using Xunit;
+
+namespace Sails.TestUtils.XUnit.Fixtures;
+
+public abstract partial class SailsFixture : IAsyncLifetime
```

Comment:
Not sure if it should be in Utils or Shared project. It is like an internal class for remoting tests.

Reply:
Not sure it will be used for ClientGenerator tests, but moved for now into Sails.Remoting